<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>couponDialog</class>
 <widget class="QDialog" name="couponDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>매장 정보</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>상호명</string>
   </property>
  </widget>
  <widget class="QLabel" name="restNameLable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>매장 주소</string>
   </property>
  </widget>
  <widget class="QLabel" name="restAddrLable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>매장 전화번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="restNumberLable">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>사업자</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownerLable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>194</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사업자 등록번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownerNumLable">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>194</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
